Fixing Grammar (#22)
polymerdevs/Quest-Into-The-Polyverse-Phase-1: Participant Instructions/Judging Criteria.md

| Criteria | 10 | 5 | 0 |
| --- | --- | --- | --- |

> |Project completion and openness|Code implements sufficient features for a functional MVP that the community can interact with. The code is open-source and properly licensed. We believe that community collaboration and licensing are permissive, and how to contribute clear and open-sourced code makes contribution easy. Further development path or potential roadmap is defined. |Code is a viable MVP but not ready to use for the community. The project indicates it is open to contributions, but lacks clear guidelines, making it difficult for contributors to understand how to contribute effectively. Licensing is clear.|Implementation is incomplete and lacks a clear development path to provide an MVP. Licensing isn’t clear or source code is closed preventing community development to MVP.|

Reviewer: ```suggestion
|Project completion and openness|Code implements sufficient features for a functional MVP that the community can interact with. The code is open-source and properly licensed. We believe in community collaboration: licensing is permissive and how to contribute is clear, to make open-source contribution easy. Additionally, the further development path or potential roadmap is defined. |Code is a viable MVP but not ready to use for the community. The project indicates it is open to contributions, but lacks clear guidelines, making it difficult for contributors to understand how to contribute effectively. Licensing is clear.|Implementation is incomplete and lacks a clear development path to provide an MVP. Licensing isn’t clear or source code is closed preventing community development to MVP.|
```
